src/app/index.js

```diff
@@ -0,0 +1,6 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+ReactDOM.render(
+  <h1>Hello, world!</h1>, document.querySelector('body')
```

Comment:
Do not render directly to the body element because other stuff can modify the content of the body tag and that will mess up your react application. (Use a `div` with an id `root`)